backend/src/ban/repository/ban.repository.ts

```diff
@@ -21,6 +21,21 @@ export class BanRepository implements IBanRepository {
     return result ? result.unbanned_date : null;
   }
 
+  async getIsPenalty(user_id: number): Promise<boolean> {
+    const result = await this.banLogRepository.findOne({
+      select: {
+        is_penalty: true,
+      },
+      where: {
+        ban_user_id: user_id,
+      },
+      order: {
+        unbanned_date: 'DESC',
+      },
+    });
+    return result.is_penalty;
```

Comment:
만약에 결과가 null일 때 is_penalty를 참조하면 예상치 못한 결과가 나타날 수 있을거 같습니다. 예를 들면 이렇게 하는 것은 어떨까여?
```suggestion
    return result ? result.is_penalty : false;
```

Reply:
좋습니다!! 감사합니다~~